### ISSUE TITLE
explanation for pickle file generation inside data folder

### ISSUE THREAD

**Issue by vinayakumarr** [renan-campos/MSA#1]
how to apply your code for other data sets? if you explain, it will be very helpful for begginners

**Comment by rcampos2029** [comment]
src/data.py extracts its train/test set by looking for files ending in '.txt' in the following directories:
data/train/pos  # For positive training examples
data/train/neg # For negative training examples
data/test         # For unlabeled test set.
If your data set can be formatted in this way, you could replace the files in the above directories and run train.py/test.py. Otherwise, you may need to modify data.py.

**Comment by vinayakumarr** [comment]
Inside temp folder I have seen so many pickle files. Theses files will be generated automatically. Becoz I am going to use other language dataset .  What actually these pickle file contains

**Comment by rcampos2029** [comment]
The vectorizer, SVM, and model weights are pickled so that they can be quickly loaded and used during cross-validation and testing. Running the train script (python train.py) will overwrite all of these pickle files.

**Comment by vinayakumarr** [comment]
I installed same verion of numy, scipy, theano,sklearn and drill but still your code generates an error. Please check the attached file for that error
![untitled](https://cloud.githubusercontent.com/assets/18217467/15799597/547b4fd6-2a7f-11e6-99f5-ec5c8da5f841.png)

**Comment by renan-campos** [comment]
Can you open a new issue for this?
